### ISSUE TITLE
Suddenly not able to submit new Testflight builds - Beta App Description is missing.

### ISSUE THREAD

**Issue by tmolitor-stud-tu** [fastlane/fastlane#28630]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description
I'm calling `fastlane run upload_to_testflight` and get an error: `[!] Beta App Description is missing. - Beta App Description is required to submit a build for external testing. (Spaceship::UnexpectedResponse)`.
I'm using a test command as minimal example here, not uploading a new build, but using the last uploaded build.
Our CI used to issue this command over here, and up until 2024-11-12 this worked flawlessly: https://github.com/monal-im/Monal/blob/develop/.github/workflows/beta.build-push.yml#L167

Am I doing something wrong? Did something change in recent versions of fastlane regarding this api?
Unfortunately I don't know the fastlane version it last worked with (could be some older version that was updated between  2024-11-12 and now).

##### Command executed
`fastlane run upload_to_testflight` (see the complete commandline in the details below).

##### Complete output when running fastlane, including the stack trace and command used

<details>
  <pre>PILOT_LOCALIZED_APP_INFO='{"default": {"feedback_email": "info@monal-im.org", "marketing_url": "https://monal-im.org/", "privacy_policy_url": "https://monal-im.org/privacy/", "description": "There has never been a better time to get into XMPP, a free public chat network no one controls or owns. Monal is a fast and user friendly way to use XMPP. Just download the app,  login or register and you are ready to chat in minutes. It looks and works the way other apps do, so there is no need to “learn XMPP” or even care what it is.\n\nNotable features:\n- Open Source\n- No Ads! Strong focus on privacy. Does not phone home and does not have software \"metrics\" \n- Does not read any personal information\n- With a direct connection to your server, your password and all other info are never sent to a third-party\n- OMEMO encrypted chat\n- Will work with corporate XMPP servers that require VPN\n- Multi user chat (MUC) support for group chats\n- Audio/Video calls\n\nImplements certain XMPP extensions intended to improve mobile communication:\n- XEP-0357: Push Notifications\n- XEP-0280: Message Carbons keep messages in synch between clients\n- XEP-0198: Stream Management to quickly reconnect\n- XEP-0199: XMPP Ping to maintain connections\n- XEP-0313: Message Archive Management to download chat history\n- XEP-0352: Client State Indication for dramatic reduction on power use\n- XEP-0363: HTTP File Upload to send images in conversations\n"},"de-DE": {"feedback_email": "info@monal-im.org", "marketing_url": "https://monal-im.org/", "privacy_policy_url": "https://monal-im.org/privacy/", "description": "Es gab noch nie einen besseren Zeitpunkt, um in XMPP einzusteigen, ein kostenloses öffentliches Chat-Netzwerk, das niemand kontrolliert oder besitzt. Monal ist ein schneller und benutzerfreundlicher Weg zur Nutzung von XMPP. Lade einfach die App herunter, melde dich an oder registriere dich, und schon kannst du in wenigen Minuten chatten.\n\nWesentliche Merkmale:\n- Open Source\n- Keine Werbung! Starker Fokus auf Privatsphäre. Ruft nicht zu Hause an und hat keine \"Messdaten\"-Software.\n- Liest keine persönlichen Informationen aus.\n- Mit einer direkten Verbindung zu deinem Server, werden dein Passwort und alle anderen Informationen niemals an Dritte gesendet.\n- OMEMO verschlüsselter Chat\n- Funktioniert mit XMPP-Servern von Unternehmen, die VPN erfordern\n- Mehrbenutzer-Chats in Gruppen oder Channels\n- Audio/Video-Anrufe\n\nImplementiert bestimmte XMPP-Erweiterungen, um die mobile Kommunikation zu verbessern.\n- XEP-0357: Push-Benachrichtigungen\n- XEP-0280: Nachrichtenkopien halten Nachrichten zwischen Clients synchron\n- XEP-0198: Stream-Management zum schnellen Wiederverbinden\n- XEP-0199: XMPP-Ping zur Aufrechterhaltung von Verbindungen\n- XEP-0313: Nachrichtenarchivverwaltung zum Herunterladen des Chatverlaufs\n- XEP-0352: Client-Statusanzeige zur drastischen Reduzierung des Stromverbrauchs\n- XEP-0363: HTTP-Dateitransfer zum Senden von Bildern in Unterhaltungen\n"},"en-US": {"feedback_email": "info@monal-im.org", "marketing_url": "https://monal-im.org/", "privacy_policy_url": "https://monal-im.org/privacy/", "description": "There has never been a better time to get into XMPP, a free public chat network no one controls or owns. Monal is a fast and user friendly way to use XMPP. Just download the app,  login or register and you are ready to chat in minutes. It looks and works the way other apps do, so there is no need to “learn XMPP” or even care what it is.\n\nNotable features:\n- Open Source\n- No Ads! Strong focus on privacy. Does not phone home and does not have software \"metrics\" \n- Does not read any personal information\n- With a direct connection to your server, your password and all other info are never sent to a third-party\n- OMEMO encrypted chat\n- Will work with corporate XMPP servers that require VPN\n- Multi user chat (MUC) support for group chats\n- Audio/Video calls\n\nImplements certain XMPP extensions intended to improve mobile communication:\n- XEP-0357: Push Notifications\n- XEP-0280: Message Carbons keep messages in synch between clients\n- XEP-0198: Stream Management to quickly reconnect\n- XEP-0199: XMPP Ping to maintain connections\n- XEP-0313: Message Archive Management to download chat history\n- XEP-0352: Client State Indication for dramatic reduction on power use\n- XEP-0363: HTTP File Upload to send images in conversations\n"},"es-ES": {"feedback_email": "info@monal-im.org", "marketing_url": "https://monal-im.org/\n", "privacy_policy_url": "https://monal-im.org/privacy/\n", "description": "Nunca hubo un mejor momento para empezar a usar XMPP, una red de pública de chat que no se encuentra bajo el control ni la propiedad de nadie. Monal es la vía rápida y amigable para usar XMPP. Solo descargue la app, ingrese o regístrese y estará listo para chatear en minutos. Se ve y se usa de la misma manera que las otras apps, así que no existe necesidad de “aprender XMPP” o preocuparse por lo que es.\n\nCaracterísticas más notables:\n- Código abierto\n- Sin publicidad! Fuerte foco en la privacidad. No envía información de uso ni \"métricas\" \n- No recaba ningún tipo de información personal\n- Con conexión directa al servidor, su contraseña y el resto de la información nunca se envían a terceros\n- Chat encriptado con OMEMO\n- Funciona con servidores corporativos de XMPP que requieren una VPN\n- Soporta chat multi usuario (MUC - Multi User Chat) para chats en grupo\n- Llamadas de Audio y Video\n\nImplementa ciertas extensiones XMPP destinadas a mejorar la comunicación movil:\n- XEP-0357: Notificaciones automáticas\n- XEP-0280: Copia sincronizada de mensajes entre los distintos clientes\n- XEP-0198: Gestión de flujo de datos para una reconexión rápida\n- XEP-0199: Ping XMPP para mantener la conexión\n- XEP-0313: Administrador de archivo de mensajes para descarga del historial de chat\n- XEP-0352: Visualización de estado del cliente para reducir dramáticamente el uso de la batería\n- XEP-0363: Carga de archivos HTTP para enviar imágenes en las conversaciones\n"},"fr-FR": {"feedback_email": "info@monal-im.org", "marketing_url": "https://monal-im.org/", "privacy_policy_url": "https://monal-im.org/privacy/", "description": "C’est le meilleur moment pour se mettre à XMPP, un réseau de chat public libre que personne ne contrôle ou possède. Monal est une application facile à utiliser pour rejoindre le réseau XMPP. Téléchargez l’application, créez un compte, et ça y est vous pouvez chatter en quelques minutes. Elle est fonctionnellement identique aux applications de chat que vous connaissez, donc il n’y a aucun besoin d’« apprendre XMPP » ou même de se préoccuper de ce que c’est.\n\nFonctionalités notables :\n- Open Source\n- Aucune pub ! Mettant l’accent sur la vie privée, Monal n’utilise aucune fonctionnalité de tracking.\n- Ne lit aucune information personnelle.\n- Avec une connexion directe à votre serveur, votre mot de passe et toutes vos autres informations ne sont jamais envoyées à un tiers.\n- Chat chiffré avec OMEMO.\n- Fonctionne avec les serveurs XMPP d’entreprises qui nécessitent un VPN.\n- Chat multi-utilisateur·ice·s grâce à MUC.\n- Appels audio/video.\n\nImplémente certaines extensions XMPP pour améliorer les communications mobiles :\n- XEP-0357: Push Notifications, pour avoir des notifications même quand l’application est fermée.\n- XEP-0280: Message Carbons, pour garder les messages synchronisés entre clients.\n- XEP-0198: Stream Management, pour se reconnecter rapidement.\n- XEP-0199: XMPP Ping, pour maintenir la connexion.\n- XEP-0313: Message Archive Management, pour récupérer l’historique des messages.\n- XEP-0352: Client State Indication, pour diminuer drastiquement la consommation énergétique.\n- XEP-0363: HTTP File Upload, pour envoyer des images, des messages vocaux ou des fichiers dans les conversations.\n"},"ro": {"feedback_email": "info@monal-im.org", "marketing_url": "https://monal-im.org/", "privacy_policy_url": "https://monal-im.org/privacy/", "description": "Nu a existat niciodată un moment mai bun pentru a utiliza XMPP, o rețea publică gratuită de chat pe care nimeni nu o controlează și nu o deține. Monal este un mod rapid și prietenos de a utiliza XMPP. Trebuie doar să descărcați aplicația, să vă autentificați sau să vă înregistrați și sunteți gata de discuții în câteva minute. Arată și funcționează la fel ca alte aplicații, deci nu este nevoie să \"învățați XMPP\" sau chiar să vă pese ce este.\n\nCaracteristici notabile:\n- Sursă deschisă\n- Fără reclame! Accent puternic pe confidențialitate. Nu transmite date altora și nu vă analizează acțiunile\n- Nu citește nicio informație personală\n- Cu o conexiune directă la serverul dvs., parola dvs. și toate celelalte informații nu sunt niciodată trimise unei terțe părți\n- Discuții criptate cu OMEMO\n- Va funcționa cu servere XMPP corporatiste care necesită VPN\n- Discuții de grup MUC\n- Apeluri audio/video\n\nImplementează anumite extensii XMPP menite să îmbunătățească comunicarea mobilă.\n- XEP-0357: Notificări push\n- XEP-0280: Message Carbons menține mesajele sincronizate între clienți\n- XEP-0198: Stream Management pentru reconectarea rapidă\n- XEP-0199: Ping XMPP pentru menținerea conexiunilor\n- XEP-0313: Message Archive Management pentru a descărca istoricul conversațiilor\n- XEP-0352: Client State Indication pentru reducerea drastică a consumului de energie\n- XEP-0363: HTTP File Upload pentru a trimite imagini în conversații\n"}}' PILOT_CHANGELOG="- Add Arabic appstore description
  - Fix some rare crashes" fastlane run --verbose upload_to_testflight api_key_path:"./key.json" team_id:"S8D843U34Y" distribute_external:true notify_external_testers:true groups:"Internal Pre-Beta Testers","Public Beta" reject_build_waiting_for_review:true submit_beta_review:true distribute_only:true app_identifier:G7YU7X7KRJ.SworIM app_platform:ios build_number:993

/opt/homebrew/Cellar/fastlane/2.225.0/libexec/gems/fastlane-2.225.0/spaceship/lib/spaceship/connect_api/api_client.rb:223:in `handle_response': [!] Beta App Description is missing. - Beta App Description is required to submit a build for external testing. (Spaceship::UnexpectedResponse)
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/gems/fastlane-2.225.0/spaceship/lib/spaceship/connect_api/api_client.rb:125:in `post'
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/gems/fastlane-2.225.0/spaceship/lib/spaceship/connect_api/testflight/testflight.rb:119:in `post_beta_app_review_submissions'
        from /opt/homebrew/Cellar/ruby/3.3.6/lib/ruby/3.3.0/forwardable.rb:240:in `post_beta_app_review_submissions'
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/gems/fastlane-2.225.0/spaceship/lib/spaceship/connect_api/models/build.rb:186:in `post_beta_app_review_submission'
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/gems/fastlane-2.225.0/pilot/lib/pilot/build_manager.rb:452:in `distribute_build'
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/gems/fastlane-2.225.0/pilot/lib/pilot/build_manager.rb:203:in `distribute'
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/gems/fastlane-2.225.0/fastlane/lib/fastlane/actions/upload_to_testflight.rb:32:in `run'
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/gems/fastlane-2.225.0/fastlane/lib/fastlane/runner.rb:263:in `block (2 levels) in execute_action'
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/gems/fastlane-2.225.0/fastlane/lib/fastlane/actions/actions_helper.rb:69:in `execute_action'
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/gems/fastlane-2.225.0/fastlane/lib/fastlane/runner.rb:255:in `block in execute_action'
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/gems/fastlane-2.225.0/fastlane/lib/fastlane/runner.rb:229:in `chdir'
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/gems/fastlane-2.225.0/fastlane/lib/fastlane/runner.rb:229:in `execute_action'
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/gems/fastlane-2.225.0/fastlane/lib/fastlane/one_off.rb:42:in `run'
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/gems/fastlane-2.225.0/fastlane/lib/fastlane/one_off.rb:22:in `execute'
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/gems/fastlane-2.225.0/fastlane/lib/fastlane/commands_generator.rb:226:in `block (2 levels) in run'
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/gems/commander-4.6.0/lib/commander/command.rb:187:in `call'
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/gems/commander-4.6.0/lib/commander/command.rb:157:in `run'
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/gems/commander-4.6.0/lib/commander/runner.rb:444:in `run_active_command'
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/gems/fastlane-2.225.0/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb:124:in `run!'
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/gems/commander-4.6.0/lib/commander/delegates.rb:18:in `run!'
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/gems/fastlane-2.225.0/fastlane/lib/fastlane/commands_generator.rb:363:in `run'
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/gems/fastlane-2.225.0/fastlane/lib/fastlane/commands_generator.rb:43:in `start'
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/gems/fastlane-2.225.0/fastlane/lib/fastlane/cli_tools_distributor.rb:123:in `take_off'
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/gems/fastlane-2.225.0/bin/fastlane:23:in `<top (required)>'
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/bin/fastlane:25:in `load'
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/bin/fastlane:25:in `<main>'
/opt/homebrew/Cellar/fastlane/2.225.0/libexec/gems/fastlane-2.225.0/spaceship/lib/spaceship/connect_api/api_client.rb:223:in `handle_response': Beta App Description is missing. - Beta App Description is required to submit a build for external testing. (Spaceship::UnexpectedResponse)
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/gems/fastlane-2.225.0/spaceship/lib/spaceship/connect_api/api_client.rb:125:in `post'
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/gems/fastlane-2.225.0/spaceship/lib/spaceship/connect_api/testflight/testflight.rb:119:in `post_beta_app_review_submissions'
        from /opt/homebrew/Cellar/ruby/3.3.6/lib/ruby/3.3.0/forwardable.rb:240:in `post_beta_app_review_submissions'
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/gems/fastlane-2.225.0/spaceship/lib/spaceship/connect_api/models/build.rb:186:in `post_beta_app_review_submission'
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/gems/fastlane-2.225.0/pilot/lib/pilot/build_manager.rb:452:in `distribute_build'
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/gems/fastlane-2.225.0/pilot/lib/pilot/build_manager.rb:203:in `distribute'
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/gems/fastlane-2.225.0/fastlane/lib/fastlane/actions/upload_to_testflight.rb:32:in `run'
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/gems/fastlane-2.225.0/fastlane/lib/fastlane/runner.rb:263:in `block (2 levels) in execute_action'
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/gems/fastlane-2.225.0/fastlane/lib/fastlane/actions/actions_helper.rb:69:in `execute_action'
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/gems/fastlane-2.225.0/fastlane/lib/fastlane/runner.rb:255:in `block in execute_action'
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/gems/fastlane-2.225.0/fastlane/lib/fastlane/runner.rb:229:in `chdir'
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/gems/fastlane-2.225.0/fastlane/lib/fastlane/runner.rb:229:in `execute_action'
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/gems/fastlane-2.225.0/fastlane/lib/fastlane/one_off.rb:42:in `run'
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/gems/fastlane-2.225.0/fastlane/lib/fastlane/one_off.rb:22:in `execute'
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/gems/fastlane-2.225.0/fastlane/lib/fastlane/commands_generator.rb:226:in `block (2 levels) in run'
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/gems/commander-4.6.0/lib/commander/command.rb:187:in `call'
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/gems/commander-4.6.0/lib/commander/command.rb:157:in `run'
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/gems/commander-4.6.0/lib/commander/runner.rb:444:in `run_active_command'
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/gems/fastlane-2.225.0/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb:124:in `run!'
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/gems/commander-4.6.0/lib/commander/delegates.rb:18:in `run!'
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/gems/fastlane-2.225.0/fastlane/lib/fastlane/commands_generator.rb:363:in `run'
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/gems/fastlane-2.225.0/fastlane/lib/fastlane/commands_generator.rb:43:in `start'
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/gems/fastlane-2.225.0/fastlane/lib/fastlane/cli_tools_distributor.rb:123:in `take_off'
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/gems/fastlane-2.225.0/bin/fastlane:23:in `<top (required)>'
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/bin/fastlane:25:in `load'
        from /opt/homebrew/Cellar/fastlane/2.225.0/libexec/bin/fastlane:25:in `<main>'</pre>
</details>

### Environment

<details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                         | Value                                                      |
| --------------------------- | ---------------------------------------------------------- |
| OS                          | 14.7                                                       |
| Ruby                        | 3.3.6                                                      |
| Bundler?                    | false                                                      |
| Git                         | git version 2.39.3 (Apple Git-146)                         |
| Installation Source         | /opt/homebrew/Cellar/fastlane/2.225.0/libexec/bin/fastlane |
| Host                        | macOS 14.7 (23H124)                                        |
| Ruby Lib Dir                | /opt/homebrew/Cellar/ruby/3.3.6/lib                        |
| OpenSSL Version             | OpenSSL 3.4.0 22 Oct 2024                                  |
| Is contained                | false                                                      |
| Is homebrew                 | true                                                       |
| Is installed via Fabric.app | false                                                      |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/                |
| Xcode Version               | 15.3                                                       |
| Swift Version               | 5.10                                                       |

### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | de_DE.UTF-8 | ✅ |
| LC_ALL   |             |   |
| LANGUAGE |             |   |

### fastlane files:

**No Fastfile found**

**No Appfile found**

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.225.0 | ✅ Up-To-Date  |

### Loaded fastlane plugins:

**No plugins Loaded**

<details><summary><b>Loaded gems</b></summary>

| Gem                             | Version      |
| ------------------------------- | ------------ |
| error_highlight                 | 0.6.0        |
| did_you_mean                    | 1.6.3        |
| syntax_suggest                  | 2.0.1        |
| public_suffix                   | 6.0.1        |
| addressable                     | 2.8.7        |
| artifactory                     | 3.0.17       |
| jmespath                        | 1.6.2        |
| aws-partitions                  | 1.1009.0     |
| aws-eventstream                 | 1.3.0        |
| aws-sigv4                       | 1.10.1       |
| aws-sdk-core                    | 3.213.0      |
| aws-sdk-kms                     | 1.96.0       |
| aws-sdk-s3                      | 1.172.0      |
| babosa                          | 1.0.4        |
| bundler                         | 2.5.23       |
| rexml                           | 3.3.9        |
| nkf                             | 0.1.3        |
| base64                          | 0.2.0        |
| CFPropertyList                  | 3.0.7        |
| colored                         | 1.2          |
| highline                        | 2.0.3        |
| commander                       | 4.6.0        |
| dotenv                          | 2.8.1        |
| emoji_regex                     | 3.2.3        |
| excon                           | 0.112.0      |
| faraday-em_http                 | 1.0.0        |
| faraday-em_synchrony            | 1.0.0        |
| faraday-excon                   | 1.1.0        |
| faraday-httpclient              | 1.0.1        |
| multipart-post                  | 2.4.1        |
| faraday-multipart               | 1.0.4        |
| faraday-net_http                | 1.0.2        |
| faraday-net_http_persistent     | 1.2.0        |
| faraday-patron                  | 1.0.0        |
| faraday-rack                    | 1.0.0        |
| faraday-retry                   | 1.0.3        |
| ruby2_keywords                  | 0.0.5        |
| faraday                         | 1.10.4       |
| faraday_middleware              | 1.2.1        |
| domain_name                     | 0.6.20240107 |
| http-cookie                     | 1.0.7        |
| faraday-cookie_jar              | 0.0.7        |
| fastimage                       | 2.3.1        |
| sysrandom                       | 1.0.5        |
| fastlane-sirp                   | 1.0.0        |
| gh_inspector                    | 1.1.3        |
| uber                            | 0.1.0        |
| declarative                     | 0.0.20       |
| trailblazer-option              | 0.1.2        |
| representable                   | 3.2.0        |
| retriable                       | 3.1.2        |
| mini_mime                       | 1.1.5        |
| jwt                             | 2.9.3        |
| multi_json                      | 1.15.0       |
| os                              | 1.1.4        |
| signet                          | 0.19.0       |
| googleauth                      | 1.8.1        |
| httpclient                      | 2.8.3        |
| google-apis-core                | 0.11.3       |
| google-apis-androidpublisher_v3 | 0.54.0       |
| google-apis-playcustomapp_v1    | 0.13.0       |
| google-cloud-env                | 1.6.0        |
| google-cloud-errors             | 1.4.0        |
| google-cloud-core               | 1.7.1        |
| google-apis-iamcredentials_v1   | 0.17.0       |
| google-apis-storage_v1          | 0.31.0       |
| rake                            | 13.2.1       |
| digest-crc                      | 0.6.5        |
| google-cloud-storage            | 1.47.0       |
| json                            | 2.7.2        |
| mini_magick                     | 4.13.2       |
| naturally                       | 2.2.1        |
| optparse                        | 0.4.0        |
| plist                           | 3.7.1        |
| rubyzip                         | 2.3.2        |
| security                        | 0.1.5        |
| simctl                          | 1.6.10       |
| terminal-notifier               | 2.0.0        |
| unicode-display_width           | 2.6.0        |
| terminal-table                  | 3.0.2        |
| tty-screen                      | 0.8.2        |
| tty-cursor                      | 0.7.1        |
| tty-spinner                     | 0.9.3        |
| word_wrap                       | 1.0.0        |
| atomos                          | 0.1.3        |
| claide                          | 1.1.0        |
| colored2                        | 3.1.2        |
| nanaimo                         | 0.4.0        |
| xcodeproj                       | 1.27.0       |
| rouge                           | 2.0.7        |
| xcpretty                        | 0.3.0        |
| xcpretty-travis-formatter       | 1.0.1        |
| set                             | 1.1.0        |
| forwardable                     | 1.3.3        |
| logger                          | 1.6.0        |
| pathname                        | 0.3.0        |
| shellwords                      | 0.2.0        |
| cgi                             | 0.4.1        |
| date                            | 3.3.4        |
| timeout                         | 0.4.1        |
| stringio                        | 3.1.1        |
| securerandom                    | 0.3.1        |
| uri                             | 0.13.1       |
| ipaddr                          | 1.2.6        |
| openssl                         | 3.2.0        |
| digest                          | 3.1.1        |
| io-nonblock                     | 0.3.0        |
| zlib                            | 3.1.1        |
| resolv                          | 0.3.0        |
| io-wait                         | 0.3.1        |
| time                            | 0.3.0        |
| open-uri                        | 0.4.1        |
| mutex_m                         | 0.2.0        |
| net-http                        | 0.4.1        |
| net-protocol                    | 0.2.2        |
| english                         | 0.8.0        |
| erb                             | 4.0.3        |
| strscan                         | 3.0.9        |
| abbrev                          | 0.1.2        |
| io-console                      | 0.7.1        |
| tempfile                        | 0.2.1        |
| delegate                        | 0.3.1        |
| fileutils                       | 1.7.2        |
| tmpdir                          | 0.2.0        |
| singleton                       | 0.2.0        |
| open3                           | 0.2.1        |
| prettyprint                     | 0.2.0        |
| pp                              | 0.5.0        |
| find                            | 0.2.0        |
| ostruct                         | 0.6.0        |
| yaml                            | 0.3.0        |
| psych                           | 5.1.2        |
</details>

*generated on:* **2024-11-23**
</details>

**Comment by ugurcanatas** [comment]
I recently had a similar issue with first name, last name and email fields for beta app information. 

```bash
The provided entity is missing a required attribute - You must provide a value for the attribute 'contactEmail' with this request - /data/attributes/contactEmail (Spaceship::UnexpectedResponse)
```

What i did was:
- App Store Connect
- Go to your app
- Click on Testflight tab
- Click on App Information link under Additional section.
- Fill out the Beta App Information & Beta App Review Information sections.
- Save

After doing that, the error went away.  I believe that App store connect behaves a bit weird for the past week. I ran into login problems in the platform. So I am assuming that there are some changes on Apple's side.

**Comment by tmolitor-stud-tu** [comment]
@ugurcanatas Sure, but I'm using fastlane to *automate* things. Having to do half of the steps *manually* defeats the purpose of fastlane.

**Comment by tmolitor-stud-tu** [comment]
@ugurcanatas Did the problems go away for future submissions, too? Having to do some manual steps *once* would be acceptable, but not if I have to do them every time I want to publish a testflight build.

**Comment by ugurcanatas** [comment]
> @ugurcanatas Did the problems go away for future submissions, too? Having to do some manual steps _once_ would be acceptable, but not if I have to do them every time I want to publish a testflight build.

Yes the information is filled only once on app store connect. Afterwards, builds were successful. But of course this is me assuming that the issue described and the issue that i came across originates from the same thing. Hopefully that works for you as well

**Comment by tmolitor-stud-tu** [comment]
@ugurcanatas unfortunately your advice didn't fix my issue :(

**Comment by tmolitor-stud-tu** [comment]
the documentation says:
> changelog	Provide the 'What to Test' text when uploading a new build

But although I set the PILOT_CHANGELOG environment variable, uploading and submitting it in one step doesn't work. But it seems to be possible to do a `skip_submission:true` upload only and then afterwards do a dedicated `distribute_only:true` step to distribute the beta build uploaded in the previous step.

**Comment by tmolitor-stud-tu** [comment]
This somehow fixed itself, closing the issue now. Thanks @ugurcanatas for your help :)

**Comment by tmolitor-stud-tu** [comment]
I'll reopen this again: @ugurcanatas' workaround fixes the issue, but the original issue is still present: fastlane should submit a beta app description automatically, if provided via `localized_app_info`.
